docs/source/model_doc/t5.mdx

```diff
@@ -263,6 +263,11 @@ print(tokenizer.batch_decode(output_sequences, skip_special_tokens=True))
 
 <a id='scripts'></a>
 
+## Performance
+
+If you'd like a faster performance, install [apex](https://github.com/NVIDIA/apex#quick-start) and then the model will automatically use `apex.normalization.FusedRMSNorm` instead of `T5LayerNorm`.
+
```

Comment:
(nit) Maybe also mention faster performance for training or inference or both. 
Also, would it make sense to add some explanation about why `FusedRMSNorm` is faster?

Reply:
> (nit) Maybe also mention faster performance for training or inference or both. 

both.

> Also, would it make sense to add some explanation about why `FusedRMSNorm` is faster?

good idea on adding an explanation

I made the adjustments - please have a look if it's addressing all the needs. Thank you!

Reply:
Looks good. Thanks for adding the details!